r, ".* ERR .*-E-PVT-0- get_temperature: sensor=GIBRALTAR_HBM_SENSOR_.* is not ready.*"
r, ".* INFO .*Failed to instantiate thermal sensor SSD_Temp: xr_sysctl_get.*status -116.*"
r, ".* INFO .*[duty_cycle_map]: illegal pwm value .*"
r, ".* INFO healthd.*Key 'TEMPERATURE_INFO|ASIC' field 'high_threshold' unavailable in database 'STATE_DB'.*"
r, ".* INFO healthd.*Key 'TEMPERATURE_INFO|ASIC' field 'temperature' unavailable in database 'STATE_DB'.*"
r, ".* INFO .*command '/usr/sbin/smartctl' failed: [116] Stale file handle.*"
r, ".* ERR swss#orchagent:.*initBufferConstants: Failed to get Maximum memory size.*"
r, ".* ERR syncd#syncd:.*la_vrf_port_common_base::get_acl_group ingress acl group not found.*"
r, ".* ERR syncd#syncd:.*la_vrf_port_common_base::get_acl_group egress acl group not found.*"
r, ".* ERR syncd#syncd:.*la_acl_key_profile_base::initialize failed to place udk for key type IPV6.*"
r, ".* ERR syncd#syncd:.*Failed to retrieve system port SAI ID for port ID .*, switch not in VOQ mode.*"
r, ".* ERR syncd#syncd:.*-E-HLD-0- la_acl_key_profile_base::initialize failed to place udk for key type IPV6.*"
r, ".* ERR syncd#syncd:.*SAI_API_ACL: Invalid or unsupported ACL match field.*"
r, ".* ERR syncd#syncd:.*SAI_API_BUFFER: get_buffer_pool_stats unknown counter 20.*"
r, ".* ERR syncd#syncd:.*SAI_API_BUFFER: get_ingress_priority_group_stats unknown counter.*"
r, ".* ERR syncd#syncd:.*SAI_API_SWITCH: genl_ctrl_resolve failed family=lb_genl_family0.*"
r, ".* ERR syncd#syncd:.*-E-HLD-0- get_dependent_objects: NULL dependee.*"
r, ".* ERR syncd#syncd:.*SAI_API_LAG: resolve_feat_over_member_ports: found port index .*"
r, ".* ERR syncd#syncd:.*SAI_API_PORT: Invalid port counter .*index.*"
r, ".* ERR syncd#syncd:.*updateSupportedBufferPoolCounters: BUFFER_POOL_WATERMARK_STAT_COUNTER: counter SAI_BUFFER_POOL_STAT_XOFF_ROOM_WATERMARK_BYTES is not supported on buffer pool .* rv: SAI_STATUS_NOT_IMPLEMENTED.*"
r, ".* ERR syncd#syncd:.*SAI_API_HOSTIF: src/sai_trap.cpp:.*: Invalid trap event code 45.*"
r, ".* ERR syncd#syncd: -E-TABLES-0- shared/src/hw_tables/lpm/lpm_core.cpp::.* update_tcam status = Leaba_Err: Out of resources: virtual la_status silicon_one::lpm_core_tcam_allocator_pacific_gb.*"
r, ".* ERR syncd#syncd: SAI_API_SWITCH: destroy object failed.. references exist to la_voq_cgm_profile_impl.*"
r, ".* ERR syncd#syncd: SAI_API_SWITCH: found.* dependencies. printing first.* entries"
r, ".* ERR syncd#syncd: SAI_API_SWITCH:.* type VOQ_SET, la_voq_set_impl.*"
r, ".* ERR syncd#syncd: message repeated.* times: \[ SAI_API_SWITCH:.* type VOQ_SET, la_voq_set_impl.*"
r, ".* ERR swss#orchagent: :- queryAattributeEnumValuesCapability: returned value .* is not allowed on SAI_DEBUG_COUNTER_ATTR_IN_DROP_REASON_LIST.*"
r, ".* ERR syncd#syncd: -E-TABLES.* Core.* TCAM update failed.*"
r, ".* ERR swss#fdbsyncd: :- readData: netlink reports an error.* on reading a netlink socket.*"
r, ".* ERR kernel:.* PortChannel.*: Device Ethernet.* is up. Set it down before adding it as a team port.*"
r, ".* ERR pmon#python3.* checkReplyType: Expected to get redis type 2 got type 3.*"
r, ".* ERR pmon#python3.* guard: RedisReply catches system_error: command.*"
r, ".* ERR syncd#syncd:.*Core .* TCAM update failed.*"
r, ".* ERR syncd#syncd:.*shared/src/hw_tables/lpm/lpm_core.cpp::.* update_tcam status = Leaba_Err: Out of resources.*failed to update TCAM.*"